### PR TITLE
reimplemented coreharuna+some commands

### PR DESCRIPTION
**Pull request by takase1121**

### CODE REVIEW

**Comment by takase1121** [comment]
more fixes pending, after im not that busy

File: commands/status.js [hunk 1]
```diff
@@ -12,18 +12,17 @@ Servers  :: ${client.guilds.size}
 Users    :: ${client.users.size}
 Memory   :: ${Math.round(process.memoryUsage().rss / 1024 / 1024)} MB
 Library  :: Eris @ 0.8.6
```
Comment (sr229) [inline commands/status.js]: Don't hardcode this. Read from your DepList

File: commands/eval.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+//eval
+exports.run = async (config, client, arg, msg) => {
+    const memusage = `Heap total: ${Math.floor(process.memoryUsage().heapTotal/1048576)} MB\n` +
+        `Heap used: ${Math.floor(process.memoryUsage().heapUsed/1048576)} MB\n` +
```
Comment (sr229) [inline commands/eval.js]: don't hardcode your memlimit, it should be what set as your current GC limit.

Comment (takase1121) [reply commands/eval.js]: hardcode? what you mean by that? btw that string is to check the mem i put it there so i can call it with eval

File: commands/status.js (hunk 1 above)
Comment (sr229) [reply commands/status.js]: Parse your package.json and parse Eris.

File: commands/eval.js (hunk 2 above)
Comment (sr229) [reply commands/eval.js]: Heap limit int. Remember Memory varies per system.

**Comment by Deivu** [comment]
Code was Updated a lot since this last commit. You may need to recheck a lot of things.

**Pull request closed without merging**